ROADMAP.md

```diff
@@ -0,0 +1,25 @@
+# Roadmap
+
+Our first priority is [Complete Feature Coverage](https://github.com/openactive/openactive-test-suite/issues/87). Below are the milestones towards achieving that.
+
+## Milestones
+
+This doc is our roadmap. It's a place to start to investigate the myriad of issues that you can contribute to. Check out the different milestones listed below, but feel free to explore the issues by label too.
+
+Please check out our [contributing guidelines](./CONTRIBUTING.md) and code of conduct to help you get started, and the [README](./README.md) for an overview if you haven't read it yet!
+
+### Milestone 1: [“core”](https://github.com/openactive/openactive-test-suite/tree/master/packages/openactive-integration-tests/test/features/core) and [“payment”](https://github.com/openactive/openactive-test-suite/tree/master/packages/openactive-integration-tests/test/features/payment) features
+
+_Expected 12th August_
```

Comment:
Perhaps:

_Aiming for completion by 12th August_

More inclusive language for those contributing?

Reply:
Done